migration/migrator/migrations/course/20251212150903_course_materials_on_calendar copy.py

```diff
@@ -0,0 +1,35 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+def up(config, database, semester, course):
```

Comment:
this should be combined with previous migration.
Also, the migration file should be renamed to be the current date (ideally the date should be close to when the PR is actually merged... to make sure migrations from different PRs are processed in merge order)